home/home4.1/src/Main.java

```diff
@@ -0,0 +1,92 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    private static Map<String, Double> ingredients = new HashMap<>();
+
+    public static void main(String[] args) {
```

Comment:
This function is to long. Let's split it to handlers and so on. For example, classic pizza can be generated with some separate methods. Some other logic of user choice handling can be also moved to functions.